tests/cli/test_cli.py

```diff
@@ -4,24 +4,17 @@
 
 def test_cli_start(run: Callable[..., RunResult]):
     """
-    Measures an average startup time and checks that it
-    does not deviate more than x seconds from 5.
+    Checks that a call to ``rasa --help`` does not take longer than 7 seconds.
     """
     import time
 
-    durations = []
+    start = time.time()
+    run("--help")
+    end = time.time()
 
-    for i in range(5):
-        start = time.time()
-        run("--help")
-        end = time.time()
+    duration = end - start
 
-        durations.append(end - start)
-
-    avg_duration = sum(durations) / len(durations)
-
-    # When run in parallel, it takes a little longer
-    assert avg_duration - 5 <= 2
+    assert duration <= 5 + 2
```

Comment:
This `5 + 2` part seems a bit weird, especially since we specify that it should take no longer than 7 seconds in the description of the test.
Maybe worth replacing with just `7`